Makefile

```diff
@@ -307,10 +307,8 @@ protoc: $(PROTOGEN) $(MOCKGEN) $(GOIMPORTS) $(PROTOC_GEN_GO) $(PROTOC_GEN_GO_GRP
 		-root=$(ROOT) \
 		-proto-out=$(PROTO_OUT) \
 		-proto-root=$(PROTO_ROOT) \
-		-protogen=$(PROTOGEN) \
 		-api-binpb=$(API_BINPB) \
-		-goimports=$(GOIMPORTS) \
-		-mockgen=$(MOCKGEN) \
+		-tool-bin-dir=$(ROOT)/$(LOCALBIN) \
```

Comment:
Seems easier to just point at a single tools dir here. Especially since proto-gen-go is picked up through PATH; not an explicit path.

Reply:
Well, `goimports` and `mockgen` are already installed without a version in their file name. Note that this is different from the ".stamp` dir where all have versions.

<img width="243" height="218" alt="Screenshot 2025-09-15 at 11 03 00 AM" src="https://github.com/user-attachments/assets/e080991b-6d00-493b-9ed1-3c85923e2420" />